index.js

```diff
@@ -1 +1,9 @@
-module.exports = 0; 
+if(typeof module !== 'undefined')
+{
+  module.exports = 0;
+}
+else if(typeof window !== 'undefined')
+{
+  // Set a global variable for browsers 
+  window.numberZero = 0;
```

Comment:
_Nitpick:_ Not sure about frontend JS conventions, but shouldn't this be `NUMBER_ZERO`?